filters/filter_1_Russian/template.txt

```diff
@@ -5,32 +5,37 @@
 !------------------ General JS API ---------------------------------------------!
 !-------------------------------------------------------------------------------!
 @include ../general_js_api.txt
+!
+!-------------------------------------------------------------------------------!
+!------------------ Common advertising networks --------------------------------!
+!-------------------------------------------------------------------------------!
+@include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/CyrillicFilters/common-sections/adservers.txt"
+!-------------------------------------------------------------------------------!
+!------------------ Common banner names ----------------------------------------!
+!-------------------------------------------------------------------------------!
+@include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/CyrillicFilters/common-sections/general_url.txt"
+!-------------------------------------------------------------------------------!
+!------------------ Common general hiding rules---------------------------------!
+!-------------------------------------------------------------------------------!
+@include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/CyrillicFilters/common-sections/general_elemhide.txt"
+!
 !-------------------------------------------------------------------------------!
 !------------------ Advertising networks ---------------------------------------!
 !-------------------------------------------------------------------------------!
-@include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/RussianFilter/sections/adservers.txt"
 @include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/RussianFilter/sections/adservers_firstparty.txt"
```

Comment:
Shouldn't there be new includes like this?
```
@include "https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/CyrillicFilters/RussianFilter/sections/adservers_firstparty.txt"
```

Reply:
done